pylint_venv.py

```diff
@@ -124,5 +124,6 @@ def inithook(venv=None, force_venv_activation=False):
     if venv is None:
         return
 
-    print(f"Using env: {venv}", file=sys.stderr)
+    if not quiet:
+        print(f"Using env: {venv}", file=sys.stderr)
```

Comment:
This is already printing to `stderr` to have this output separate from `stdin` where other tools should read from ... makes me wonder a bit why this output was a problem. Still it won't hurt to be able to disable it.